provider/caleta/model_mapper.go

```diff
@@ -198,3 +201,30 @@ func getTransactionsMapper(ctx context.Context, r *WalletWinBody) pam.GetTransac
 		}, nil
 	}
 }
+
+func roundTransactionsMapper(roundTransactions *[]roundTransaction) *[]pam.RoundTransaction {
+	if roundTransactions != nil {
+		var roundTx []pam.RoundTransaction
+		for _, t := range *roundTransactions {
+			amt := toPamAmount(t.Payload.Amount)
+			txID := t.Payload.TransactionUUID
+			isGameOver := t.Payload.RoundClosed
+			tx := pam.RoundTransaction{
+				ProviderTransactionId: &txID,
+				CashAmount:            &amt,
+				IsGameOver:            &isGameOver,
+				TransactionDateTime:   &t.ClosedTime, // TODO: Or is it CreatedTime?
```

Comment:
Hehe good question, we should give i a try and test it later

Reply:
🤞